modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-test/src/testFunctional/tests/CPReportsadmin.testcase

```diff
@@ -121,8 +121,6 @@ definition {
 			locator1 = "ReportsAdminViewReportEntry#INFO_MESSAGE",
 			value1 = "Processing report. This may take several minutes.");
 
-		Pause(locator1 = "180000");
```

Comment:
Hi @chaowang-123 !
About this pause, I think it's being used to force the portal wait until the Report is processed. 
Could you, please, try to use something like 

`while (IsElementPresent(ReportsAdminViewReportEntry#INFO_MESSAGE) && (maxIterations = "10")) {
			Refresh();
}`
Thank you!

Reply:
ok, I will update it, thanks!